### PR TITLE
(tlg1443)-tlg1443.tlg004-tlg010 to tlg1443.tlg001

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Test conversion and combination of these files.

### CODE REVIEW

File: data/tlg1443/tlg001/__cts__.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg1443" xml:lang="grc" urn="urn:cts:greekLit:tlg1443.tlg004">
+  <ti:title xml:lang="lat">Epistulae vii genuinae</ti:title>
```
Comment (lcerrato) [inline data/tlg1443/tlg001/__cts__.xml:2]: unchanged

Comment (AlisonBabeu) [reply data/tlg1443/tlg001/__cts__.xml:2]: That is weird I changed it on my computer. I will try again.

Comment (AlisonBabeu) [reply data/tlg1443/tlg001/__cts__.xml:2]: No wait now, I changed it from 004 to 001, that was what I wanted to do, the build is just not finished.

**Comment by AlisonBabeu** [comment]
hi @lcerrato re the new pull request with the new work by this author, I'm pretty sure I used Ignatius of Antioch in all the files but will double check after this goes through.

**Review by lcerrato — APPROVED** [review]

**Pull request merged**